build/pkgs/networkx/distros/conda.txt

```diff
@@ -1 +1 @@
-networkx<3.0,>=2.4
+networkx<3.2,>=2.4
```

Comment:
```suggestion
networkx>=2.4
```
I suggest this change which depends on #36544.

Reply:
This PR is already on Volker's branch. I'd suggest to merge it into #36544 and make the change there